cmake/utils/ifm3d-dpkg-deps.py.in

```diff
@@ -1,8 +1,8 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 # -*- python -*-
 
 #
-# Copyright (C)  2017 Love Park Robotics, LLC
+# Copyright (C)  2017 ifm electronic, gmbh
 #
 # Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
I do recommend the usage of the SPDX header for licensing purposes. Which makes it mus easier for automated tool to scan for the license.

Reply:
To be addressed in https://github.com/ifm/ifm3d/issues/216